pandas/tests/io/test_parquet.py

```diff
@@ -931,6 +931,13 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    def test_read_zero_chunked_array(self, pa):
```

Comment:
may want to parameterize over several dtypes here (with the empty array)

Reply:
Does this mean I should do it this way?
```suggestion
    def test_read_zero_chunked_array(self, pa):
        df = pd.DataFrame(
            {
                "a": pd.array([], dtype="Int64"),
                "b": pd.array([], dtype="UInt8"),
                "c": pd.array([], dtype="string"),
                "d": pd.array([], dtype="boolean"),
            }
        )
        check_round_trip(df, pa)
```